elide-contrib/elide-dynamic-config-helpers/src/main/resources/elideTableSchema.json

```diff
@@ -0,0 +1,435 @@
+        },
+        "category": {
+          "title": "Measure group category",
+          "description": "Category for grouping",
+          "type": "string"
+        },
+        "hidden": {
+          "title": "Hide/Show measure",
+          "description": "Whether this metric is exposed via API metadata",
+          "type": "boolean",
+          "default": false
+        },
+        "readAccess": {
+          "title": "Measure read access",
+          "description": "Read permission for the metric.",
+          "type": "string",
+          "default": "Allow All"
+        },
+        "definition": {
+          "title": "Metric definition",
+          "description": "The definition of the metric",
+          "type": "string"
+        },
+        "type": {
+          "title": "Field type",
+          "description": "The data type of the field",
+          "type": "string",
+          "enum": [
+            "TIME",
+            "INTEGER",
+            "DECIMAL",
+            "MONEY",
+            "TEXT",
+            "COORDINATE",
+            "BOOLEAN"
+          ],
+          "default": "INTEGER"
+        }
+      },
+      "required": [
+        "name",
+        "definition"
+      ],
+      "additionalProperties": false
+    },
+    "dimension": {
+      "title": "Dimension",
+      "description": "Dimensions represent labels for measures. Dimensions are used to filter and group measures.",
+      "type": "object",
+      "properties": {
+        "name": {
+          "title": "Dimension name",
+          "description": "The name of the dimension. This will be the same as the POJO field name.",
+          "type": "string",
+          "pattern": "^([A-Za-z]*_?[A-Za-z]*)*$"
+        },
+        "description": {
+          "title": "Dimension description",
+          "description": "A long description of the dimension.",
+          "type": "string",
+          "default": "The dimension name"
+        },
+        "category": {
+          "title": "Dimension group category",
+          "description": "Category for grouping dimension",
+          "type": "string"
+        },
+        "hidden": {
+          "title": "Hide/Show dimension",
+          "description": "Whether this dimension is exposed via API metadata",
+          "type": "boolean",
+          "default": false
+        },
+        "readAccess": {
+          "title": "Dimension read access",
+          "description": "Read permission for the dimension.",
+          "type": "string",
+          "default": "Allow All"
+        },
+        "definition": {
+          "title": "Dimension definition",
+          "description": "The definition of the dimension",
+          "type": "string",
+          "default": ""
+        },
+        "type": {
+          "title": "Dimension field type",
+          "description": "The data type of the dimension field",
+          "type": "string",
+          "enum": [
+            "text",
+            "coordinate",
+            "number",
```

Comment:
Number is no longer a type - we now have integer and decimal or something along those lines.  

Check out ValueType in Elide.